src/SignalRServiceExtension/Constants.cs

```diff
@@ -8,6 +8,7 @@ internal static class Constants
         public const string AzureSignalRConnectionStringName = "AzureSignalRConnectionString";
         public const string AzureSignalREndpoints = "Azure:SignalR:Endpoints";
         public const string AzureSignalRHubProtocol = "Azure:SignalR:HubProtocol";
+        public const string AzureSignalRNewtonsoftCamelCase = "Azure:SignalR:HubProtocol:Newtonsoft:CamelCase";
```

Comment:
Would this config too deep?

Reply:
`Azure:SignalR:HubProtocol` accepts `SystemJson` and `NewtonsoftJson`, is there relationship with this config?

Reply:
`AzureSignalRNewtonsoftCamelCase` would take effect only when `Azure:SignalR:HubProtocol=NewtonsoftJson`.